--- utility.py	2018-08-28 15:06:36.699153400 -0500
+++ utility2.py	2018-08-28 15:04:54.977000000 -0500
@@ -725,7 +725,8 @@
         if abs_uri.startswith(pfx):
             abs_uri = sub + abs_uri[len(pfx):]
     if 0 > abs_uri.find(':'):
-        abs_uri = os.path.realpath(abs_uri)
+        # abs_uri = os.path.realpath(abs_uri)
+        abs_uri = os.path.normpath(os.path.join(os.path.dirname(parent_uri), uri))
     return abs_uri
 
 
